tasks/main.yml

```diff
@@ -21,20 +21,20 @@
 
 # will be replaced with symlink
 - name: Ensure logs directory is not present.
-  file: state=absent path={{symfony2_project_root}}/releases/{{symfony2_project_release}}/app/logs
+  file: state=absent path={{symfony2_project_root}}/releases/{{symfony2_project_release}}/{{symfony2_project_relative_root}}/app/logs
```

Comment:
If the relative root is filled by default with "/", the path will read 
```{{symfony2_project_root}}/releases/{{symfony2_project_release}}///app/logs```
I'd suggest to use empty relative path var and comment to use trailing slash.